webviz_subsurface/plugins/_volumetric_analysis/utils/table_and_figure_utils.py

```diff
@@ -209,3 +210,136 @@ def add_line(
 
         # update margin to make room for the labels
         figure.update_layout({"margin_t": 100})
+
+
+class VolumeWaterFallPlot:
+    def __init__(
+        self,
+        barnames: List[str],
```

Comment:
underscore more consistent with the other inputs
```suggestion
        bar_names: List[str],
```